lib/cisco_node_utils/cmd_ref/bgp_neighbor_af.yaml

```diff
@@ -94,27 +109,33 @@ max_prefix_threshold:
   default_value: ~
 
 max_prefix_warning:
-  default_value: ~
+  cli_ios_xr:
+    default_value: 75
```

Comment:
This seems very weird given that the max_prefix_warning getter for NXOS returns either `true` or `nil` - is it really correct that `warning-only` means `75` on XR?